CHANGELOG.md

```diff
@@ -1,14 +1,26 @@
-# 5.x
+## Unreleased
+
+### Added
+
+* Added first pass of _variants_. Variants different configurations of testing the same code.
+  * Add support for _variant dependencies_.
+    * wct already supports loading dependencies from your `bower_components` directory, mapping them to `../` in your code. You can now add variant dependency directories named like `bower_components-foo`. When these are detected, tests will then run separately for each such dependency directory, mapping `../` appropriately. See README for more details.
+
+### Removed
```

Comment:
Does this also remove wct auto-providing client-side dependencies via the npm package, ie: requiring the installation of the Bower package?